### ISSUE TITLE
Script to create files needed for OH service publishing from a docker-compose.yml file 

### ISSUE THREAD

**Issue by SanjeevKGupta** [open-horizon/anax#2642]
In order to on-board an application, developed using docker-compose, to open-horizon - a conversion utility will be highly productive. Even creating a boiler-plate/skeleton output would be 90% there and then hand-tweak rest of it as needed. Something like

```
hzn convert docker-compose.yml -o json horizon-files
```

**Comment by bmpotter** [comment]
I agree this would be useful. The thing that concerns me is the docker-compose yml file has tons of options. We couldn't support all of them, and documenting which we support and which we don't would be messy and probably confusing to the user. I don't see the core anax team having bandwidth to implement this and maintain it over time. So i think this would require someone else who wants to champion this and maintain it over time.

**Comment by bmpotter** [comment]
A couple of suggestions when implementing this:
- The utility help should enumerate which docker-compose features it currently supports, and which docker-compose.yml file versions it supports
- When parsing the yml file, it should try to recognize features it doesn't yet support and give warnings about that to stderr
- Support the option to write the service definition to stdout, so it could be piped into `hzn exchange service publish -f-`

**Comment by dlarson04** [comment]
Closing as won't do. Not a customer requirement now.
